e2e/fragments/directions.js

```diff
@@ -6,13 +6,13 @@ module.exports = {
       direction: {
         title: `#${party}_${index}_directionType`,
         description: `#${party}_${index}_directionText`,
-        dueDate: `#${party}_${index}_dateToBeCompletedBy`,
       },
     };
   },
 
   async enterDate(party, dueDate, index = 0) {
-    await I.fillDateAndTime(dueDate, this.fields(party, index).direction.dueDate);
+    const dateSelector = `(//*[contains(@class, "collection-title")])[${ index + 1 }]/parent::div//*[contains(@class,"form-date")]`;
```

Comment:
is it worth overwriting the `this.fields(party, index).direction.dueDate ` value with this new xpath? to follow the existing pattern?